tests/GslCore.Tests/testPromTermLen.fs

```diff
@@ -0,0 +1,85 @@
+﻿module testPromTermLen
+/// Test #promoterlen #terminatorlen work
+
+open NUnit.Framework
+open constants
+open commonTypes
+open pragmaTypes
+open Amyris.ErrorHandling
+
+/// location of test gslc_lib fixtures (depends on .net tooling version)
+let testLibDir1 = @"../../../../TestGslcLib"
+let testLibDir2 = @"../../../../../TestGslcLib"
```

Comment:
I've had success using `AppDomain.CurrentDomain.BaseDirectory` to make this deterministic, but if this is due to dotnetcore vs. mono then ouch.